server/controllers/Users.js

```diff
@@ -0,0 +1,47 @@
+import models from '../database/models';
+// import { generateToken, serverResponse, serverError } from '../helpers';
+import { serverResponse, serverError } from '../helpers';
+import hashPassword from '../helpers/passwords';
+
+const { User } = models;
+
+/**
+ * @export
+ * @class User
```

Comment:
this should be `Users`

Reply:
> Why is there no confirm password field?

We asked the PO for the fields to use and she said password field is okay. There was no confirm password field in the fields she approved of. I followed that